nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/AbstractTestAnnotationNotification.java

```diff
@@ -136,10 +140,12 @@ public void testCommentManagerType() {
         assertEquals(getType(), commentManager.getClass());
     }
 
-    protected void captureAndVerifyAnnotationEventNotification(Supplier<DocumentModel> supplier,
+    protected DocumentModel captureAndVerifyAnnotationEventNotification(Supplier<DocumentModel> supplier,
```

Comment:
we can changes the returned type to `void` as the returned document is not used or i am missed some things

Reply:
Not anymore indeed. I used it with the first version of the test